scalafmt-tests/src/test/resources/scala3/RenameImportExport.stat

```diff
@@ -0,0 +1,75 @@
+
+<<< single import
+import scala   as  java
+>>>
+import scala as java
+<<< select import
+import java.util.List   as  `J-List`
+>>>
+import java.util.List as `J-List`
+<<< multiple imports
+import java.util.{List   as  `J-List`, LinkedList as Linked}
+>>>
+import java.util.{List as `J-List`, LinkedList as Linked}
+<<< multiple imports multiline
+maxColumn = 20
+===
+import java.util.{List   as  `J-List`, LinkedList as Linked}
+>>>
+import java.util.{
+  List as `J-List`,
+  LinkedList as Linked
+}
+<<< redundant braces
+import java.util.{List   as  `J-List`}
```

Comment:
what if it had been `.{List => JList}`? do you need to check for `as` vs `=>`? or even, if the language is known to be scala3, rewrite `=>` to `as`?

Reply:
We need to make sure it's `as`. I forgot about that one. Rewriting `=>` to `as` would probably be a nice feature, but maybe we can follow up with it if there's interest?

Reply:
Fixed and add an additional point to the issue.